### ISSUE TITLE
HELP! All Area Pokemon Dots are Missing in PLA 

### ISSUE THREAD

**Issue by amargagaminghd** [kwsch/pkNX#336]
I just downloaded the latest pknx "#20230518.1 • Update dependencies"

when I click Show field editors => map viewer => select specific pokemon => it won't show the dot maps
![Capture33+](https://github.com/kwsch/pkNX/assets/29346112/8b882e1d-6480-48db-828e-5013d130b1a3)

**Comment by duckdoom4** [comment]
That's a bug, we'll have to look into it and get it fixed

**Comment by amargagaminghd** [comment]
@duckdoom4 Do you have an old version of PKNX? Last year version was working but the latest is not

**Comment by duckdoom4** [comment]
On the page where you downloaded this version, you can also find older builds

**Comment by amargagaminghd** [comment]
@duckdoom4 it's only a few weeks ago version 
![sdfsdfsdf](https://github.com/kwsch/pkNX/assets/29346112/f86fbb2a-0bfc-40f3-86b9-cb855146ba6a)

**Comment by duckdoom4** [comment]
Yeah, doesn't that older version work?

**Comment by amargagaminghd** [comment]
@duckdoom4 the 2022 version build I believe

**Comment by duckdoom4** [comment]
If non of those work you'll have to wait for a bugfix

**Comment by RevanXVIV** [comment]
Wait is someone working on the PLA Randomizer settings? Last I remember it was unsupported. If it's being worked on that's great news!

**Comment by duckdoom4** [comment]
> Wait is someone working on the PLA Randomizer settings? Last I remember it was unsupported. If it's being worked on that's great news!

No, what gave you that idea? :)

**Comment by RevanXVIV** [comment]
> > Wait is someone working on the PLA Randomizer settings? Last I remember it was unsupported. If it's being worked on that's great news!
> 
> No, what gave you that idea? :)

I can't tell if you're being serious or not.

**Comment by duckdoom4** [comment]
> > > Wait is someone working on the PLA Randomizer settings? Last I remember it was unsupported. If it's being worked on that's great news!
> > 
> > No, what gave you that idea? :)
> 
> I can't tell if you're being serious or not.

Yes, that was a serious question. I don't quite understand why you think someone is working on it? (No one is FYI)

**Comment by RevanXVIV** [comment]
> > > > Wait is someone working on the PLA Randomizer settings? Last I remember it was unsupported. If it's being worked on that's great news!
> > > 
> > > 
> > > No, what gave you that idea? :)
> > 
> > 
> > I can't tell if you're being serious or not.
> 
> Yes, that was a serious question. I don't quite understand why you think someone is working on it? (No one is FYI)
> 
> (And I don't mean to sound rude or anything)

It was the smiley face at the end. I've seen people use that on occasion to subtly confirm if they are working on something. My bad. lol

**Comment by amargagaminghd** [comment]
still not working for the latest build

**Comment by duckdoom4** [comment]
I'm working on fixing up the bugs that occured due to the internal structure update. This will be one of the bugs I fix. Expect to see a fix soon

**Comment by IvyAnnette** [comment]
I think I'm having the same problem, but the pokemon in the catching tutorial (originally rowlet, cindy, and oshawott) aren't spawning for me. I'm getting the dots on the map though.

**Comment by RevanXVIV** [comment]
Wait is PLA getting support again?
